CSS/styles.css

```diff
@@ -0,0 +1,215 @@
+/* --------------------------- Base styles --------------------------- */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Arial', sans-serif;
+    -webkit-font-smoothing: antialiased; /* do I need this? */
+    background: #FFF;
+    color:#222;
+}
+
+ul {
+    list-style: none;
+}
+
+/* 
+h1, h2, h3, h3, h4 {
+
+}
+*/
+
+a {
+    color: #222;
+    text-decoration: none;
+}
+
+
+p {
+    margin: 0.5rem 0;
+}
+
+img {
+    width: 100%;
+}
+
+.content-wrap {
+    width: 90%;
+    margin: 0 auto;
+}
+
+/* Buttons */
+.btn {
+    display: inline-block;
+    cursor: pointer;
+    border: 1px solid #DDD;
+    border-radius: 4px;
+    background-color: #FFF;
+}
+
+.btn:hover {
+
+}
+
+.btn-rounded {
+	border-radius: 4px;
+}
```

Comment:
We should talk linters. One of the things experienced web developers are very picky about is formatting and style conventions. Having mixed tabs and spaces is a no-no from a style perspective (but perfectly valid code).

Reply:
Interesting. It looks the same width in my editor, so I'm not sure what happened there. I've copied and pasted a tab from another line. Hopefully that will fix it. I'll try installing some of those extensions that we talked about.